src/assignments/html-me-somthing.rst

```diff
@@ -0,0 +1,338 @@
+Assignment #4: HTML Me Someting
+================================
+
+You’ve learned a bit of HTML and some CSS, but you have likely only used
+it in bits and pieces so far, adding or modifying content in exercises
+or pre-existing files. Here, you are going to take another step forward
+by building an entire page from scratch. You will also get some practice
+using Git.
+
+There are two parts to this exercise, one focused on HTML and another
+focused on CSS. HTML makes up the *structure and content* of web pages,
+while CSS dictates the *visual style*.
+
+Best practices dictate that *content* and *style* should be kept as separate as
+possible. To that end, we will build the HTML portion of our page first,
+and afterwards we will add a few styles with CSS. This will help us avoid
+creating HTML elements simply to change the style of our page.
+
+Sections:
+----------
+
+#. :ref:`Getting Started <getting-started>`
+#. :ref:`Getting to Work <getting-to-work>`
+#. :ref:`Submitting Your Work <submitting-your-work>`
+
+.. _getting-started:
+
+Getting started
+----------------
+
+First, follow the steps below to create a folder for your project and
```

Comment:
Unnecessary "First,". Change to "Follow the steps below to create a folder for your project..."

Reply:
Fixed.